include/TrustWalletCore/TWBinanceSigner.h

```diff
@@ -19,5 +19,6 @@ struct TWBinanceSigner;
 /// Signs a transaction.
 TW_EXPORT_STATIC_METHOD
 TW_Binance_Proto_SigningOutput TWBinanceSignerSign(TW_Binance_Proto_SigningInput input);
-
+TWData *_Nonnull TWBinanceMessage(TW_Binance_Proto_SigningInput data);
```

Comment:
you need to add `TW_EXPORT_STATIC_METHOD` annotation and follow the naming, otherwise codegen will be failed

Reply:
Thank you, added
could you please explain a bit about the codegen mechanism? does it only for iOS/Android?
In go code, I can include these declarations via include the header files and it works. I just don't understand what I will break if I miss them.

Reply:
https://developer.trustwallet.com/wallet-core/contributing#c-headers